web/rainmaker/dev-packages/egov-tradelicence-dev/src/ui-utils/commons.js

```diff
@@ -382,6 +374,8 @@ export const applyTradeLicense = async (state, dispatch, activeIndex) => {
 
       let accessories = get(queryObject[0], "tradeLicenseDetail.accessories");
       let tradeUnits = get(queryObject[0], "tradeLicenseDetail.tradeUnits");
+      const tradeSubTypeNew = get( state, "screenConfiguration.preparedFinalObject.DynamicMdms.TradeLicense.tradeUnits.tradeSubType", []);
```

Comment:
Please give a meaningful and standard name to a variable. instead of using 'new' suffix?